postgresql-11.5/src/bin/psql/commandbteq.c

```diff
@@ -214,6 +219,8 @@ exec_command(const char *cmd,
         status = exec_command_set(scan_state, active_branch);
     } else if (strcasecmp(cmd, "quit") == 0 || strcasecmp(cmd, "q") == 0) {
         status = exec_command_quit(scan_state, active_branch);
+    } else if (strcmp(cmd, "c") == 0 || strcmp(cmd, "connect") == 0) {
+        status = exec_command_connect(scan_state, active_branch);
```

Comment:
Why do we need this?

Reply:
it was added to allow us to join database to use tables

Reply:
After executing LOGON command you should already be able to use the database to which you are connected and all its tables.

The LOGON command is supposed to connect to the database and nothing else.